src/pages/Homepage.js

```diff
@@ -152,31 +157,44 @@ function Homepage() {
     }
 
     async function setGradeForState(stateId) {
-        const result = await fetchStateGrade(stateId);
+        let result = await fetchStateGrade(stateId);
+        if (result === undefined) {
+            result = 'API call failed'
+        }
         $('.card-gun-grade')
             .children('p.detail-header').text(`Giffords State Gun Law Grade:`);
         $('.card-gun-grade')
             .children('p.detail-text').text(result);
     }
 
     async function setOwnershipDataForState(stateId) {
-        const results = await fetchGunOwnershipLevels(stateId);
-        $('.card-household')
```

Comment:
Looks like you accidentally deleted the header text for Percentage of Households with a Gun